core/src/test/java/org/pentaho/platform/dataaccess/datasource/api/MetadataServiceTest.java

```diff
@@ -678,5 +680,14 @@ public void createNewByteArrayInputStream() throws Exception {
     assertNotNull( stream );
   }
 
+  @Test
+  public void extractXmiFromZipTest() throws Exception {
```

Comment:
for test names follow the naming convention in the class. In the git diffs you can see "testImportMetadataFromTemp()" 
"test" + <NAME-OF-FUNCTION-UNDER-TEST> so this class should be"testExtractXmiFromZip"